R/onload.R

```diff
@@ -1,6 +1,27 @@
+#' @importFrom cachem cache_disk
+#'
+#' @importFrom memoise memoise
 .onLoad <- function(libname, pkgname) {
-    # a <- Sys.getenv("DATAVERSE_SERVER")
-    # if(a == "") {
-    #     Sys.setenv("DATAVERSE_SERVER" = "dataverse.harvard.edu")
-    # }
+  # a <- Sys.getenv("DATAVERSE_SERVER")
+  # if(a == "") {
```

Comment:
Probably delete this comment together, which seems to be a remnant of an old version. We now have an explicit error for missing `server` argument